packages/abstract-core/src/native/version_control.rs

```diff
@@ -57,6 +69,11 @@ pub enum ExecuteMsg {
     RemoveModule { module: ModuleInfo, yank: bool },
     /// Add new modules
     AddModules { modules: Vec<ModuleMapEntry> },
+    /// Approve or decline modules
+    ApproveOrDeclineModule {
```

Comment:
Can we make this accept a `Vec`? If users can add modules in batches we also want to be able to approve/reject in batches